todo_app/templates/index.html

```diff
@@ -12,7 +12,21 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+          {% for x in listOfItems %}
+            <tr>
```

Comment:
I think `tr` and `th` tags only work within a `table` tag. I think you want to replace both these tags with an `li` tag.

https://www.w3schools.com/tags/tag_tr.asp
https://www.w3schools.com/tags/tag_li.asp